pipeline/tests/asm.rs

```diff
@@ -454,3 +454,10 @@ fn test_failing_assertion() {
     let i = [];
     verify_asm(f, slice_to_vec(&i));
 }
+
+#[test]
+fn connect_no_witgen() {
```

Comment:
I think witgen works, actually? Because you constrained `w` in a way that lets witgen figure out the values even when copy constraints are ignored.

Reply:
I mean sure. The name just means that this does not require witgen, because the involved columns are all fully defined.

Reply:
The point of this test is to run this against the polygon verifier.